### PR TITLE
Extensions docs 811 pgvector

### PR DESCRIPTION
**Pull request by piano35-edb**
Add pgvector extension docs.

### CODE REVIEW

File: advocacy_docs/pg_extensions/pgvector/installing.mdx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  To set up the repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+
+## Install the package
+
+The syntax for the package install command is:
+
+```shell
+# For SLES, CentOS, RHEL and its derivatives
+sudo <package-manager> -y install edb-<postgres><postgres_version>-pg-vector1
```
Comment (djw-m) [inline advocacy_docs/pg_extensions/pgvector/installing.mdx:36]: Package name is wrong.

pgvector0

File: advocacy_docs/pg_extensions/pgvector/installing.mdx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+# For Debian and Ubuntu
+sudo <package-manager> -y install edb-<postgres><postgres_version>-pg-vector-1
```
Comment (djw-m) [inline advocacy_docs/pg_extensions/pgvector/installing.mdx:39]: Package name is wrong.

pgvector-0

File: advocacy_docs/pg_extensions/pgvector/installing.mdx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+ | PostgreSQL                   | pg         |
+ | EDB Postgres Advanced Server | as         |
+ | EDB Postgres Extended Server | postgresextended |
+
+- `<postgres_version>` is the version of Postgres you're using.
+
+For example, to install pgvector for EDB Postgres Advanced Server 15 on a RHEL 8 platform:
+
+```shell
+sudo dnf -y install edb-as15-pg-vector1
```
Comment (djw-m) [inline advocacy_docs/pg_extensions/pgvector/installing.mdx:65]: See above commants on package name.

**Review by djw-m — CHANGES_REQUESTED** [review]
Package names need fixing

**Comment by djw-m** [comment]
You also have to change index.mdx.in to insert pgvector in the right place in the left hand side bar.

You will then need to run the extensions generator to update the extensions page itself which will make the changes to the extensions.json file and the navigation changes stick.

**Review by djw-m — APPROVED** [review]
LGTM

**Comment by piano35-edb** [comment]
Add docs for extension pgvector.

**Pull request merged**